src/gui_common/dialogs/CustomConfirmationDialog.cs

```diff
@@ -139,7 +143,17 @@ private void UpdateLabel()
         if (dialogLabel == null)
             throw new SceneTreeAttachRequired();
 
-        dialogLabel.Text = TranslationServer.Translate(dialogText);
+        var text = new StringBuilder(100);
+
+        if (CenterText)
+            text.Append("[center]");
```

Comment:
Instead of centering the text like this, I think it would be so much better to use the text alignment on the label to do this. This is because rich text is expensive operation to parse and should not be used when it is avoidable.